functionary/ui/templates/admin/base_site.html

```diff
@@ -0,0 +1,81 @@
+    <link rel="stylesheet"
+          href="{% static 'css/fontawesome-all-5.15.4.min.css' %}"
+          type="text/css"/>
+    <link rel="stylesheet"
+          href="{% static 'css/functionary.css' %}"
+          type="text/css"/>
+    <link rel="stylesheet" href="{% static 'css/admin.css' %}" type="text/css"/>
+{% endblock extrahead %}
+<!-- Header -->
+{% block branding %}
+    <div class="columns">
+        <div class="columns column is-12">
+            <div class="column is-8">
+                <h1 id="site-name">
+                    <div onclick="location.href={% url 'admin:index' %}" class="admin-header">
+                        <img src="{% static 'functionary_f_logo.png' %}"
+                             alt="F"
+                             width="64"
+                             height="64"/>
+                        <span class="functionary-title">unctionary administration</span>
+                    </div>
+                </h1>
+            </div>
+            <div class="column is-4">
+                <div id="user-tools" class="top-right-container has-text-right">
+                    {% if user.is_authenticated %}
+                        <div>
+                            <p>
+                                <span>{% translate "Welcome," %}</span>&nbsp;{{ user }}
+                            </p>
+                        </div>
+                        <div>
+                            {% if user.is_active and user.is_staff %}
+                                {% url 'django-admindocs-docroot' as docsroot %}
+                                {% if docsroot %}
+                                    <p>
+                                        <a href="{{ docsroot }}">{% translate "Documentation" %}</a> /
+                                    </p>
+                                {% endif %}
+                            {% endif %}
+                            {% if user.has_usable_password %}
+                                <p>
+                                    <span class="icon"><i class="fa fa-key"></i></span>
+                                    <a href="{% url 'admin:password_change' %}">{% translate "Change password" %}</a>
+                                </p>
+                            {% endif %}
+                            <p>
+                                <form id="logout-form" method="post" action="{% url 'admin:logout' %}">
+                                    {% csrf_token %}
+                                    <span class="icon has-text-danger"><i class="fa fa-door-open"></i></span>
+                                    <button type="submit">{% translate "Log out" %}</button>
+                                </form>
+                            </p>
+                        </div>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock branding %}
+{% block usertools %}
+{% endblock usertools %}
+{% block nav-sidebar %}
```

Comment:
There's a fly-out panel on the left side of the list and detail pages for models that broke somehow.  A javascript error shows up in the console and when you click the >> on the collapsed panel nothing happens.  I don't know what's causing that, but I'm guessing it's something in these template customizations.